### ISSUE TITLE
Content disappears in modals (scrolling content) when reducing browsers height

### ISSUE THREAD

**Issue by alexnicolaisen** [Semantic-Org/Semantic-UI#6645]
I'm using Semantic-UI 2.4, and have noticed a problem when using modals with scrolling content.

I've created a short video, showing the problem a couple of times: http://www.youtube.com/watch?v=sLEURC_i2Dc

When reducing the height of the browser, some of the content in the modal disappears when the scrollbar of the browser appears. Sometimes textfields, text and dropdowns just go away... I can't see any changes to the HTML in the console, so I just don't know why some of the content disappears from time to time.

I've just been able to recreate the issue in Chrome, and not Edge.

Anyone with a fix/workaround here?

Thanks!

**Comment by hammy2899** [comment]
How can't you create it in a JSFiddle? If you can't recreate it in a standalone context like a JSFiddle we will have to assume its caused by something on your site.

**Comment by alexnicolaisen** [comment]
Yes...I have this script running: "Hide random content when there are two or more scrollbars on the page"....but only for Chrome and not Edge :)

I thought you could be interrested in taking a look at it, but no problem, I'll just change "scrolling content" to "content". 

Sorry to waste your time, but do you know of anyone good at Semantic-UI that can be hired to take a look at it? I like the "scrolling content" design, and will pay someone to make it work :)

**Comment by alexnicolaisen** [comment]
<3

Are you able to recreate it in jsFiddle?

It is so random here, and I have a dropdown, inside a table inside a grid, inside an accordion, inside a modal, so I don't see what's the cause...

**Comment by alexnicolaisen** [comment]
@hammy2899 - I'm not alone with this at least, so even if it's site-related, someone should take a look at it :)

**Comment by hammy2899** [comment]
Ok, when I can get some time I will see if I can replicate and find the problem. Until then if anyone else can reproduce this feel free to share.
